### ISSUE TITLE
Not able to install pcf dev on my laptop

### ISSUE THREAD

**Issue by bsridhar123** [vmware-archive/pcfdev#45]
Hi,

I just got access to download pcf dev from pivotal network.

I have a few questions on getting started.

What would be the minimum RAM  requirements and storage space required for it.

When i tried to install it is just taking a lot of time , it says around 5 to 6 hrs for downloading the dependencies and it got stopped many times in between.

So wanted to know if i have the required infrastructure as required for using it.

Thanks,
Sridhar

**Comment by sclevine** [comment]
Hi @bsridhar123,

We recommend that you allow the VM to consume 4 GB of RAM on your host system to improve performance and to leave room for applications and services. That said, I am able to run PCF Dev with the VM limited to 2 GB of RAM with no issues. Most users have a host system with at least 8 GB of RAM.

The PCF Dev Vagrantfile downloads a 4 GB box file, which requires at least 20 GB of local disk space to run. The CloudFront-hosted box download takes ~12 minutes on our machines in New York, Chicago, and Michigan. We've heard that it takes about that long to download from London as well. Are you downloading PCF Dev using a slow internet connection?

Hope we can get you up and running!
Stephen

**Comment by bsridhar123** [comment]
Thanks for your response.

I was able you install it successfully on my laptop.

I also deployed a sample java web app to it.

I will now explore service binding feature to bind the app to available services from marketplace or to services like mongodb which i already have running in my laptop. Would be more interested in trying that out and also scaling the apps. Not sure if i can do that with this minimalistic cloud foundary installation.

Anyway thanks for you and your teams effort  in providing us with a environment with which we can explore and learn a lot.

Thanks,
Sridhar
